src/RichTextEditor/FormattingToolbar/HyperlinkModal.js

```diff
@@ -95,6 +96,7 @@ const HyperlinkModal = React.forwardRef(({ ...props }, ref) => {
     // If the form is just opened, focus the Url input field
     if (props.showLinkModal) {
       setOriginalSelection(editor.selection);
+      setApplyStatus(refHyperlinkTextInput.current.props.defaultValue);
```

Comment:
let's use a boolean instead of the string, so you can instead do `setApplyStatus(!!refHyperlinkTextInput.current.props.defaultValue)`